framework/framework.go

```diff
@@ -48,31 +51,42 @@ type Config struct {
 	//
 	// NOTE this is deprecated since the CRD concept is the successor of the TPR.
 	TPR tpr.Interface
+
+	// Settings.
+	BackOffFactory func() backoff.BackOff
 }
 
 // DefaultConfig provides a default configuration to create a new operator
 // framework by best effort.
 func DefaultConfig() Config {
 	return Config{
 		// Dependencies.
-		BackOffFactory: nil,
 		Informer:       nil,
-		InitCtxFunc:    nil,
 		Logger:         nil,
 		ResourceRouter: nil,
 		TPR:            nil,
+
+		// Settings.
+		BackOffFactory: func() backoff.BackOff {
+			return backoff.WithMaxTries(backoff.NewExponentialBackOff(), resourceRetries)
+		},
```

Comment:
Please use the default implementation that is already there. See https://github.com/giantswarm/operatorkit/blob/f21e8578ed653172c0236a1ad7a516409ed125f5/framework/backoff_factory.go. We may want to even remove the file then as well.

Reply:
Done.